code/modules/gear_presets/uscm_ship.dm

```diff
@@ -565,6 +565,13 @@
 	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/general/large(new_human), WEAR_R_STORE)
 	new_human.equip_to_slot_or_del(new /obj/item/device/binoculars/range(new_human), WEAR_L_HAND)
 
+
+/datum/equipment_preset/uscm_ship/so/inspection // Used by /datum/emergency_call/inspection_hc
+	name = "USCM Inspection - Staff Officer (SO)"
+	role_comm_title = "Insp"
+
+/datum/equipment_preset/uscm_ship/so/inspection/load_gear(mob/living/carbon/human/new_human)
+	new_human.equip_to_slot_or_del(new /obj/item/device/radio/headset/almayer/highcom(new_human), WEAR_L_EAR)
```

Comment:
This change means this is all they get:
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/03056e6b-76d8-4465-8ae9-571bc80a0fa6)
Just a headset